src/twig/Commands/DefaultCommand.cs

```diff
@@ -51,16 +51,25 @@ public sealed class Settings : CommandSettings
             [Description("Print information after compressing each file")]
             public bool Verbose { get; set; }
 
+            [CommandOption("--advise")]
+            [Description("Find the best compression level given a file and duration")]
+            public int Duration { get; set; }
```

Comment:
Please provide an example in the [Description()]

i.e. [Description("Find the best compression level given a file and duration\nExample: twig --advise filepath 2000\nDuration is in milliseconds.")]

Reply:
not sure how to make it work in that order. I was able to do it only with filepath 2000 --advise.

Reply:
Ok, then don't worry about it. Was just hoping we could simplify a little